app/services/katello/pulp/server.rb

```diff
@@ -17,8 +17,8 @@ def self.config(url, user_remote_id)
             :debug => true
           },
           :cert_auth => {
-            :ssl_client_cert => ::Cert::Certs.ssl_client_cert(use_admin_as_cn_pulp_cert: true),
-            :ssl_client_key => ::Cert::Certs.ssl_client_key(use_admin_as_cn_pulp_cert: true)
+            :ssl_client_cert => '/etc/pki/katello/certs/pulp-client.crt',
+            :ssl_client_key => '/etc/pki/katello/private/pulp-client.key'
```

Comment:
I hard coded this for now to allow removal of the setting, the real change here is to drop all of the Pulp 2 related code paths from the code base and that seemed a more monumental task. I think this is "dead" code.

Reply:
Hardcoding is fine for now. I think it would be good though to have a redmine to reference so we can go back later on and remove the hard code.

Reply:
I wrote up https://projects.theforeman.org/issues/35006 -- would that suffice? How do you want it referenced?